talentlink-connector/processes/rest/Application.p.json

```diff
@@ -50,7 +50,7 @@
         "queryParams" : {
           "query" : "java.net.URLEncoder.encode(\"{applications(first: 10, after: 0, where: {id_eq:\"+in.applicationId+\"}) {id}}\", \"utf-8\")",
           "stream" : "",
-          "api_key" : "ivy.var.get(\"talentlink-connector.apikey\")"
+          "api_key" : ""
```

Comment:
can we remove that "api_key"? Or is it automatically added by the inscription mask after selecting the openapi-resource?
- if we can, just remove it, to not confuse the users